Stripe/UI/STPPaymentCardTextField.m

```diff
@@ -403,6 +403,55 @@ - (STPCardParams *)card {
     return c;
 }
 
+- (void)setCard:(nonnull STPCardParams *)card {
+    if (card.number.length) {
```

Comment:
Like, ideally you could reset some of these values by passing an empty card object.